lib/tokenize.cpp

```diff
@@ -3108,6 +3108,14 @@ void Tokenizer::simplifyArrayAccessSyntax()
     }
 }
 
+void Tokenizer::simplifyParameterVoid()
+{
+    for (Token* tok = list.front(); tok; tok = tok->next()) {
+        if (Token::Match(tok, "%name% ( void )"))
```

Comment:
spontanously it feels like we should check if tok is "sizeof". Would that cause unwanted "side effects"?

Reply:
Ah yea I could do that. I was just doing what was done originally in 2.7.

Reply:
Done